src/applications/appeals/shared/validations/date.js

```diff
@@ -30,5 +34,59 @@ export const isInvalidDateString = (year, day, month, dateString) => {
   );
 };
 
-export const hasErrorParts = (errorParts, invalidDate) =>
-  errorParts.month || errorParts.day || errorParts.year || invalidDate;
+export const hasdatePartErrors = (datePartErrors, invalidDate) =>
+  datePartErrors.month ||
+  datePartErrors.day ||
+  datePartErrors.year ||
+  invalidDate;
+
```

Comment:
Will not refactor `dateErrorMssgs` because of a slight difference in 995's date file.
>    } else if (hasErrorDate) {
>     errors.addError(errorMessages.invalidDate);